Stripe/STPAddress.m

```diff
@@ -202,6 +202,14 @@ - (BOOL)containsRequiredShippingAddressFields:(PKAddressField)requiredFields {
     return containsFields;
 }
 
+- (BOOL)containsRequiredSEPADebitFields {
+    return (self.line1.length > 0
+            && self.city.length > 0
+            && self.country.length > 0
+            && [STPPostalCodeValidator stringIsValidPostalCode:self.postalCode
```

Comment:
The API currently requires postal_code, and will error if it's missing. I've made a task to look into this – we can update postal code behavior in a future PR.